src/history/historyTracker.ts

```diff
@@ -9,7 +9,7 @@
  *
  * Undo/Redo will advance forward or backwards through Steps.
  */
-import DiffMatchPatch = require('diff-match-patch');
+import * as DiffMatchPatch from 'diff-match-patch';
```

Comment:
in esm output import = require is being treated as non valid (only allowed to import types) and stripped out.

![image](https://user-images.githubusercontent.com/1210596/64585140-60694e00-d34c-11e9-9c26-5d46eda62f6e.png)